engine/gamesys/src/gamesys/components/comp_camera.cpp

```diff
@@ -190,63 +229,54 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    static bool PostRenderScriptSetViewProjectionMsg(CameraComponent* camera, const dmMessage::URL& camera_url)
+    {
+        dmGameSystemDDF::SetViewProjection set_view_projection;
+        set_view_projection.m_View       = camera->m_View;
+        set_view_projection.m_Projection = camera->m_Projection;
+
+        dmGameObject::Result go_result = dmGameObject::GetComponentId(camera->m_Instance, camera->m_ComponentIndex, &set_view_projection.m_Id);
+        if (go_result != dmGameObject::RESULT_OK)
+        {
+            dmLogError("Could not send set_view_projection because of incomplete component.");
+            return false;
+        }
+
+        dmMessage::URL receiver;
+        dmMessage::ResetURL(&receiver);
+        dmMessage::Result result = dmMessage::GetSocket(dmRender::RENDER_SOCKET_NAME, &receiver.m_Socket);
+        if (result != dmMessage::RESULT_OK)
+        {
+            dmLogError("The socket '%s' could not be found.", dmRender::RENDER_SOCKET_NAME);
+            return false;
+        }
+
+        dmMessage::Post(&camera_url, &receiver, dmGameSystemDDF::SetViewProjection::m_DDFDescriptor->m_NameHash,
+            0, (uintptr_t)dmGameSystemDDF::SetViewProjection::m_DDFDescriptor, &set_view_projection, sizeof(dmGameSystemDDF::SetViewProjection), 0);
+        return true;
+    }
```

Comment:
This is here for backwards compatibility only, every camera is already available from the render script